client/src/components/Marquee/AddButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import add from '../../images/add.svg'
+import {
+  AddTicket,
+  Add,
+  AddImage,
+  AddLeftCutout,
+  AddRightCutout,
+} from './styled'
+
+const AddButton = () => {
+  return (
+    <AddTicket>
+      <AddLeftCutout />
+      <Add to="/">
+        <AddImage src={add} alt="add" />
```

Comment:
You can also just use an `img` tag directly. You should consider only creating a style component when you really need to, for example when there are styles that you would normally define with css.

However with an `img` tag, you can give it `width` directly
```
<img src ={addImagepath} width="30">
```

Reply:
The other reason to avoid this pattern is because you've defined a styled Component called `AddImage`, but you're passing it a prop for the image. 
But you should remember that props are usually meant to be arguments to a specific use case. However, the fact that you named the Component `AddImage` implies that this component will always be an `AddImage`, so it doesn't really need a prop for the image.
But if you called this Componente `CustomImage` or `ResponsiveImage` or `ImageContainer` or whatever, then passing it an image prop makes a lot of sense.